quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/VaultPropertyMappers.java

```diff
@@ -8,26 +8,45 @@ private VaultPropertyMappers() {
     public static PropertyMapper[] getVaultPropertyMappers() {
         return new PropertyMapper[] {
                 builder()
-                        .from("vault-file-path")
-                        .to("kc.spi-vault-files-plaintext-dir")
-                        .description("If set, secrets can be obtained by reading the content of files within the given path.")
+                        .from("vault")
+                        .description("Enables a vault provider.")
+                        .expectedValues("file", "hashicorp")
```

Comment:
can we add the value to the description? for sure only file, as we don't want to advertize the other right now. 

right now i wouldn't know from reading ./kc.sh build -h which ones are available, until i'd use the command with wrong values. Only then it'd state e.g. `Invalid value for option '--vault': foo. Expected values are: file, hashicorp`

Reply:
As discussed before, I have created this one already https://github.com/keycloak/keycloak/issues/9699.

Reply:
ah ok, then.